File: Python/easy/leap/leap.py
def leap_year(year):
    """Given a year, report if it is a leap year

    :param year: int - year to report

    return True if the year is a leap year, False otherwise
    """

    result = False
    if year % 4 == 0:
        result = True
    if year % 100 == 0:
        result = False
    if year % 400 == 0:
        result = True
    return result

File: C++/easy/leap/leap.cpp
#include "leap.h"

namespace leap {
  bool is_leap_year(int year) {
    bool result = false;
    if (year % 4 == 0) result = true;
    if (year % 100 == 0) result = false;
    if (year % 400 == 0) result = true;
    return result;
  }
}  // namespace leap
